css/types/image.json

```diff
@@ -105,6 +105,293 @@
+                  },
+                  {
+                    "prefix": "-moz-",
+                    "version_added": "3.6",
+                    "notes": [
+                      "Since Firefox 42, the prefixed version of gradients can be disabled by setting <code>layout.css.prefixes.gradients</code> to <code>false</code>.",
+                      "Considers <code>&lt;angle&gt;</code> to start to the right, instead of the top. I.e. it considered an angle of <code>0deg</code> as a direction indicator pointing to the right."
+                    ]
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "49"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "44",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "layout.css.prefixes.webkit",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
+                "firefox_android": [
+                  {
+                    "version_added": "16",
+                    "notes": "Before Firefox 36, gradients weren't applied on the pre-multiplied color space, leading to shades of grey unexpectedly appearing when used with transparency."
+                  },
+                  {
+                    "prefix": "-moz-",
+                    "version_added": "4",
+                    "notes": [
+                      "Since Firefox 42, the prefixed version of gradients can be disabled by setting <code>layout.css.prefixes.gradients</code> to <code>false</code>.",
+                      "Considers <code>&lt;angle&gt;</code> to start to the right, instead of the top. I.e. it considered an angle of <code>0deg</code> as a direction indicator pointing to the right."
+                    ]
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "49"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "44",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "layout.css.prefixes.webkit",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
+                "ie": {
+                  "version_added": "10",
+                  "notes": "Internet Explorer 5.5 through 9.0 supported gradients via a proprietary filter: <code><a href='http://msdn.microsoft.com/en-us/library/ms532997(VS.85,loband).aspx'>filter: progid:DXImageTransform.Microsoft.Gradient()</a></code>."
```

Comment:
I think this `<a>` should be changed to point to: https://developer.mozilla.org/docs/Web/CSS/-ms-filter#Gradient

Reply:
Good catch. The original link was added before we created the relevant MDN page. Fixed.